common/changes/@itwin/core-backend/RealityDataAccessIntegrationTests_2022-01-10-20-32.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/core-backend",
+      "comment": "Add assets used byr reality data integration tests.",
```

Comment:
Change to empty string. Changelogs are for end users. End users don't care about assets for integration tests.

Reply:
Done